lib/serve.js

```diff
@@ -51,6 +52,15 @@ const serve = (watchPath, options) => new Promise((resolve, reject) => {
 
           form.on('file', (field, file) => {
             files.push([field, file]);
+            var data = {
+              'aframe': {
```

Comment:
yeah, I'd double check if there's an `aframe` key already in the package.json and check that values don't get wiped out accidentally. (Per comment above.)